test/e2e/specs/demo-content-with-keyboard.test.js

```diff
@@ -0,0 +1,345 @@
+/**
+ * Node dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import uuid from 'uuid/v4';
+
+/**
+ * Internal dependencies
+ */
+import {
+	newPost,
+	pressWithModifier,
+	META_KEY,
+	pressTimes,
+	//getEditedPostContent,
+} from '../support/utils';
+
+async function uploadImageInTheMediaLibrary( assetFileName ) {
```

Comment:
Might be a handy utility in general! 🤔

Reply:
True, but this one is very keyboard specific. I mean it could be considered generic but we need to include the whole block and not just assume the media library is open. I think it's fine to keep this one internal to this test for now.